README.md

```diff
@@ -21,7 +21,7 @@ Or build a jar file by yourself and run:
 
 ```shell
 ./gradlew shadowJar
-java -jar build/libs/quick-start-java-0.1.2-SNAPSHOT-all.jar -h <host> -db <dbname> -u <username> -p <password>
+java -jar build/libs/quick-start-java-0.1.3-SNAPSHOT-all.jar -h <host> -db <dbname> -u <username> -p <password>
```

Comment:
nit: is it possible to remove version part from output name so we don't need to change this every time